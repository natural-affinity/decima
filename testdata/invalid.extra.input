decima -e abc 100
