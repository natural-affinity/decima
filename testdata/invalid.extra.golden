main.go:46: invalid extra amount: key: "--extra" failed type conversion: strconv.ParseFloat: parsing "abc": invalid syntax
